src/wp-content/themes/twentytwentyfour/functions.php

```diff
@@ -0,0 +1,97 @@
+	 *
+	 * @return void
+	 * @since Twenty Twenty-Four 1.0
+	 *
+	 */
+	function twentytwentyfour_block_styles() {
+		/**
+		 * The wp_enqueue_block_style() function allows us to enqueue a stylesheet
+		 * for a specific block. These will only get loaded when the block is rendered
+		 * (both in the editor and on the front end), improving performance
+		 * and reducing the amount of data requested by visitors.
+		 *
+		 * See https://make.wordpress.org/core/2021/12/15/using-multiple-stylesheets-per-block/ for more info.
+		 */
+		wp_enqueue_block_style(
+			'core/button',
+			array(
+				'handle' => 'twentytwentyfour-button-style-outline',
+				'src'    => get_template_directory_uri() . '/assets/css/button-outline.css',
+			)
+		);
+
+		/**
+		 * Add the `path` data to our stylesheet.
+		 *
+		 * This will let WordPress determine the best loading strategy for the stylesheet:
+		 * Small stylesheets will get inlined, while larger stylesheets will be loaded separately.
+		 *
+		 * See https://make.wordpress.org/core/2021/07/01/block-styles-loading-enhancements-in-wordpress-5-8/#inlining-small-assets for more info.
+		 */
+		wp_style_add_data( 'twentytwentyfour-button-style-outline', 'path', get_theme_file_path( 'assets/css/button-outline.css' ) );
+
+		register_block_style(
+			'core/details',
+			array(
+				'name'         => 'arrow-icon-details',
+				'label'        => __( 'Arrow icon', 'twentytwentyfour' ),
+				/*
+				 * Styles for the custom Arrow icon style of the Details block
+				 * https://github.com/WordPress/twentytwentyfour/issues/46
+				 */
+				'inline_style' => '.is-style-arrow-icon-details{padding-top:var(--wp--preset--spacing--10);padding-bottom: var(--wp--preset--spacing--10);border-bottom: 1px solid rgba(255, 255, 255, 0.20);}.is-style-arrow-icon-details summary{list-style-type:"\2193\00a0\00a0\00a0";}.is-style-arrow-icon-details[open] > summary{list-style-type:"\2192\00a0\00a0\00a0";}',
+			)
+		);
+		register_block_style(
+			'core/post-terms',
+			array(
+				'name'         => 'pill',
+				'label'        => __( 'Pill', 'twentytwentyfour' ),
+				/*
+				 * Styles variation for post terms
+				 * https://github.com/WordPress/gutenberg/issues/24956
+				 */
+				'inline_style' => '.is-style-pill a,.is-style-pill span:not([class], [data-rich-text-placeholder]){display:inline-block;background-color:var(--wp--preset--color--base-2);padding:0.375rem 0.875rem;border-radius:var(--wp--preset--spacing--20);}.is-style-pill a:hover{background-color:var(--wp--preset--color--contrast-3);}',
+			)
+		);
+		register_block_style(
+			'core/list',
+			array(
+				'name'         => 'checkmark-list',
+				'label'        => __( 'Checkmark', 'twentytwentyfour' ),
+				/*
+				 * Styles for the custom checkmark list block style
+				 * https://github.com/WordPress/gutenberg/issues/51480
+				 */
+				'inline_style' => 'ul.is-style-checkmark-list{list-style-type:"\2713";}ul.is-style-checkmark-list li{padding-inline-start:1ch;}',
```

Comment:
```suggestion
				'inline_style' => '
					ul.is-style-checkmark-list {
						list-style-type: "\2713";
					}
					ul.is-style-checkmark-list li {
						padding-inline-start: 1ch;
					}',
```

This is likely not a blocker, and probably a block theme best practice, but these `inline_style` arguments are unreadable. Personally, I'd love it if these were formatted in a more human-readable way. I guess that would mean something like this. But I don't love this either.

Same is true for pattern and template files. I think we should be making them as readable as possible, which we do a good job of.

Reply:
yes, I agree with this, I'm fine with the compressed version if it's a very simple one-line rule, but that is not the case and like you said, it's not readable as it is.

Reply:
I also recognize that expanding these to be more readable is a cosmetic change. So if there needs to be a larger discussion to establish a standard, I think it's fine to leave these as one liners for now.

Reply:
Just saw there was an update here. Much better!